### ISSUE TITLE
Adding annexin A5 antibody

### ISSUE THREAD

**Issue by Fabien-Maury** [obophenotype/human-phenotype-ontology#10400]
**Preferred term label:**
annexin A5 antibody positivity

**Synonyms**
annexin V antibody positivity|ANXA5 antibody positivity

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against human annexin A5.
(PMID:15652780|PMID:9067499)

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:536|ORPHA:80

**Your nano-attribution (ORCID)**
0000-0003-2854-0167

**Comment by pnrobinson** [comment]
@Fabien-Maury 
Adding new term: 
- ID: HP:6000646
- label: Anti-annexin A5 antibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:15652780|PMID:9067499
- def: The presence of autoantibodies (immunoglobulins) in the blood circulation that react against annexin A5.
- comment: 
- orcid: orcid:0000-0003-2854-0167
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10400
- synonyms:

**Comment by Fabien-Maury** [comment]
Hello,

it seems I failed to notice the already existing "Anti-annexin-V antibody positivity" ([HP:0034101](https://hpo.jax.org/app/browse/term/HP:0034101)), which I believe is the same as the new "annexin A5 antibody positivity" (HP:6000646) I've asked for.

Perhaps both of them could be merged with one being a synonym of the other ?

References:

- https://en.wikipedia.org/wiki/Annexin_A5
- https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3312215/#:~:text=Annexin%20A5%20(AnxA5%3B%20a%20protein,that%20express%20phosphatidylserine%20(1).
